packages/foam-vscode/src/features/janitor.ts

```diff
@@ -80,11 +91,9 @@ async function runJanitor(foam: Foam) {
   // Apply Text Edits to Non Dirty Notes using fs module just like CLI
 
   const fileWritePromises = nonDirtyNotes.map(note => {
-    processedFileCount += 1;
-
     let heading = generateHeading(note);
     if (heading) {
-      console.log("fs.write heading " + note.path + " " + note.title);
+      console.log("fs.write heading " + note.path + " " + note.id);
```

Comment:
what's this `console` statement needed for? debug purposes, info, or something else?

Reply:
It was already there. I am happy to remove them. I don't see why they need to be there permanently.